3.2/Dockerfile

```diff
@@ -11,9 +11,6 @@ MAINTAINER SoftwareCollections.org <sclorg@redhat.com>
 #  * $MONGODB_DATABASE - Name of the database to create
 #  * $MONGODB_ADMIN_PASSWORD - Password of the MongoDB Admin
```

Comment:
These comments are there from first commit. Personally I agree with you, so if there is no reason from OpenShift why there were added... it could be removed.

To note: other images contains similar comments, so it could be good to be consistent.